src/coins.cpp

```diff
@@ -162,6 +162,7 @@ void CCoinsViewCache::SetBestBlock(const uint256 &hashBlockIn) {
 
 bool CCoinsViewCache::BatchWrite(CCoinsMap &mapCoins, const uint256 &hashBlockIn) {
     assert(!hasModifier);
+
```

Comment:
While this is a formatting change I'd agree with, please keep formatting change in their own PR. Consider this: if for some reason PV needs to be reverted, we may end up with a conflict here for no reason, which would make the revert much easier.

The reason I'm bringing easy revert here is that you have 2 ways to handle risk: taking measure to make failing rarer, or taking measures to make failing cheaper. In most engineering fields, the former is usually preferable: crashing a building or an airplane is VERY expensive. In software, with the right set of practices, 2 can be very much preferable as it allows to move faster.

Making things easier to revert is one of these practices. in practice, it translate by very focused diffs rather than huge set of changes in their own branches to be merged all at once. Each one of these small change can be reverted individually if there is a problem with it, with little risk of conflict.